src/utils/address.cljs

```diff
@@ -68,7 +68,7 @@
     (let [first-part-of-public-key (subs public-key 0 3)
           ellipsis                 "..."
           public-key-size          (count public-key)
-          last-part-of-public-key  (subs public-key (- public-key-size 6) public-key-size)
+          last-part-of-public-key  (subs public-key (- public-key-size 5) public-key-size)
```

Comment:
hmm, this seems like a notable change. Is this a request from designers? surely this will have knock on effects to other screens etc 🤔

Reply:
@J-Son89 updated the function to take a second optional argument `last-n`

Reply:
just to be sure, you verified this was correct with the designers too?

Reply:
@J-Son89 so it should be just the last 5 characters always